README.rst

```diff
@@ -0,0 +1,34 @@
+Yola API Python Client
+======================
+
+Usage
+-----
+
+.. code:: python
+
+    from yolapy.services import Yola
+
+
+    yola = Yola(
+        url='https://wl.qa.yola.net/',
+        auth=('username', 'password'))
+
+    yola.create_user(
+        email='test@example.com',
+        name='Jane',
+        surname='Doe',
+        partner_id='WL_YOLA',
+        preferences={'name='value'})
```

Comment:
`fixed'=completed':in` https://github.com/yola/yolapy/commit/792cd56182462769891822206896d10de49bf872